Build-FFmpeg.ps1

```diff
@@ -0,0 +1,137 @@
+param(
+
+    [ValidateSet('x86', 'x64', 'ARM', 'ARM64')]
+    [string[]] $Platforms = ('x86', 'x64', 'ARM', 'ARM64'),
+
+    <#
+        Example values:
+        14.1
+        14.2
+        14.16
+        14.16.27023
+        14.23.27820
+    #>
+    [version] $VcVersion = '14.1',

```

Comment:
This has a good explanation.
The platform toolset is just a simplification for the compiler collection.
Example:
`v141` corresponds to the latest installed `14.1X` collection series (and also, to the latest `19.1X` compiler version.
There can potentially be multiple compiler collections for the same series within the same VS installation:
`14.2`: `14.20`, `14.23.27820`, etc.

The `vcvarsall.bat` script expects the compiler version (formatted as a dotted version) instead of the `v14X` string. (See the `-vcvars_ver` parameter, line 73).

This way, using `14.1` or `14.2` is perfectly equivalent to the v-prefixed names, while giving the user more control in case they want to use a very specific toolset version (which for this type of project they certainly might).

Moreover, the script defaults to the `v141` equivalent, so this is something users would very infrequently care about.

Reply:
Yikes! I managed to forget adding that argument to the .BAT invokation.
This explains why the "raw" VC version didn't make sense. Please check next commit.

Reply:
Ok I understand. If I put 14.2 and the system has 14.23.27820, it will take 14.23.27820 right? Then I am fine with this. 

But comment should then be improved, to make clear that this selects both the compiler version and platform toolkit. And maybe something like  "To get 'v141' Platform Toolkit and compiler, use '14.1', to get 'v142', use '14.2'".

Reply:
Udpated comment.